applications/adv_networking_bench/python/CMakeLists.txt

```diff
@@ -17,4 +17,9 @@
 # to set dependencies on adequate operators
 # add_custom_target(advanced_networking_bench_python ALL
 #   DEPENDS advanced_network_python
-# )
\ No newline at end of file
+# )
+
+install(
+  FILES main.py
+  DESTINATION adv_networking_bench/python
```

Comment:
Not sure yet about `DESTINATION` here, suggest we revisit in https://github.com/nvidia-holoscan/holohub/pull/506. Not necessarily a blocker.

Reply:
@tbirdso I think we are in agreement that this is good right?

Reply:
No, outcome from our discussion was that apps should install under `bin`:
```
holohub/
  ├── install-<arch>/
    ├── bin
        ├── myapp.py
```

Having an app directory `install/adv_networking_bench/<language>` is nonstandard and would not generalize well.

Reply:
I don't think that would work as some application configuration might have the same names. Why is it bad to pub it in bin/adv_networking_bench/python as we have for HSDK examples?

Reply:
>  some application configuration might have the same file names (config.yaml).

Good point.

> Why is it bad to put it in bin/adv_networking_bench/python as we have for HSDK examples?

Let's do that. It may complicate planned HAP support a bit more but I think it is the best compromise.